crates/hyperqueue/src/client/commands/job.rs

```diff
@@ -32,6 +32,22 @@ pub struct JobCancelOpts {
     pub selector_arg: SelectorArg,
 }
 
+#[derive(Parser)]
+pub struct JobCatOpts {
+    /// Select job(s) to display
+    pub job_selector_arg: SelectorArg,
+
+    /// Select task(s) of a job to display
+    pub task_selector_arg: SelectorArg,
+
+    /// Display stdout, stderr
+    #[clap(long)]
```

Comment:
Let's not combine `stdout`/`stderr` to make it simpler (otherwise we would have to distinguish between them). As stated in the issue, let the user select either `stdout` or `stderr` using a subcommand.